action-server/actions/answers.py

```diff
@@ -0,0 +1,23 @@
+from pydantic import BaseModel
+from typing import List
+
+import requests
+
+ANSWERS_PATH = "answers"
+QUESTION_KEY = "question"
+
+
+class QaResponse(BaseModel):
```

Comment:
I think the name should be more explicit (QuestionAnsweringResponse), same for protocol below.